### PR TITLE
CSS outline follows border-radius in Firefox 88

### PR DESCRIPTION
**Pull request by mirunacurtean**
A checklist to help your pull request get merged faster:
- [x] Summarize your changes

Updating FF and FF android columns version and adding notes "From Firefox 88 onwards, the standard outline property will follow the shape of border-radius."

- [x] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)

https://bugzilla.mozilla.org/show_bug.cgi?id=1694146

- [x] Link to related issues or pull requests, if any
- Fixes https://github.com/mdn/content/issues/3453

### CODE REVIEW

File: css/properties/outline-offset.json [hunk 1]
```diff
@@ -16,11 +16,18 @@
               "version_added": "15"
             },
             "firefox": {
-              "version_added": "1.5"
-            },
-            "firefox_android": {
-              "version_added": "4"
-            },
+              "version_added": "88",
+              "notes": "From Firefox 88 onwards, the standard outline property will follow the shape of border-radius."
```
Comment (ddbeck) [inline css/properties/outline-offset.json:20]:
```suggestion
              "version_added": "1.5",
              "notes": "Before Firefox 88, an outline does not follow the shape of <code>border-radius</code>."
```

**Review by ddbeck — CHANGES_REQUESTED** [review]
Thanks for this PR, @mirunacurtean. Several suggestions for you, with explanations in `outline.json`. Let me know if you have any questions!

File: css/properties/outline-color.json [hunk 2]
```diff
@@ -17,17 +17,24 @@
             },
             "firefox": [
               {
-                "version_added": "1.5"
+                "version_added": "88",
+                "notes": "From Firefox 88 onwards, the standard outline property will follow the shape of border-radius."
```
[the remark this answers is not in the document]
Comment (mirunacurtean) [reply css/properties/outline-color.json:21]: Alright, At the moment I don't know how to undo just that so I'll just update those rows to contain the previous information and push this out along with all the other suggested changes for this PR. Thanks!

File: css/properties/outline.json [hunk 3]
```diff
@@ -17,18 +17,24 @@
             },
             "firefox": [
               {
-                "version_added": "1.5",
-                "notes": "Firefox includes absolutely positioned elements inside the outline (see <a href='https://bugzil.la/687311'>bug 687311</a>)."
+                "version_added": "88",
+                "notes": "From Firefox 88 onwards, the standard outline property will follow the shape of border-radius."
```
[the remark this answers is not in the document]
Comment (mirunacurtean) [reply css/properties/outline.json]: I'm taking the expert opinion here into account :) I've updated this based on the suggestions.

File: css/properties/outline.json [hunk 4]
```diff
@@ -17,18 +17,24 @@
...
                 "version_removed": "3.6"
               }
             ],
-            "firefox_android": {
-              "version_added": "4"
-            },
+            "firefox_android": [
+              {
+                "version_added": "88",
+                "notes": "From Firefox 88 onwards, the standard outline property will follow the shape of border-radius."
```
[the remark this answers is not in the document]
Comment (mirunacurtean) [reply css/properties/outline.json]: Done in next commit.

**Comment by mirunacurtean** [comment]
@ddbeck I hope I incorporated all suggestions successfully after the latest corrections (initially I used the same version_added for FF desktop and android and that was incorrect, so I've updated all in all files). Checks are still running so fingers crossed. 
Sorry for the messiness of these updates!

**Review by ddbeck — CHANGES_REQUESTED** [review]
There are a few things left to fix here:

1. Some mis-indentation is causing CI to fail. If you run `npm run fix` and commit the results, it should correct this automatically.
2. Please delete `yarn.lock`. We don't use it.
3. I left a line comment highlight a spot I missed on the last go-around.

Thanks!

File: css/properties/outline-offset.json [hunk 5]
```diff
@@ -16,10 +16,12 @@
               "version_added": "15"
             },
             "firefox": {
-              "version_added": "1.5"
+              "version_added": "88",
```
[the remark this answers is not in the document]
Comment (mirunacurtean) [reply css/properties/outline-offset.json]: Applied in next commit.

**Comment by mirunacurtean** [comment]
@ddbeck I think this is finally ready for another look.

**Review by ddbeck — APPROVED** [review]
Looks good. Thank you! 🎉

**Pull request merged**